security/nbde_tang_server_operator/nbde-tang-server-operator-release-notes.adoc

```diff
@@ -0,0 +1,44 @@
+//NBDE Tang Server Operator Release Notes
+:_mod-docs-content-type: ASSEMBLY
+[id="nbde-tang-server-operator-release-notes"]
+= NBDE Tang Server Operator release notes
+:context: nbde-tang-server-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The following release notes track the development of the Security Profiles Operator in the OpenShift Container Platform.
+
+* link:https://access.redhat.com/errata/RHEA-2023:120658-03[RHEA-2023:120658-03 - Release of the NBDE Tang Server Operator 1.0.0]
+
+// the customer-facing link must be updated
+
+
+////
+
+[id="nbde-tang-server-operator-release-notes-1-0-0"]
+== NBDE Tang Server Operator 1.0.0
+
+The following advisory is available for the NBDE Tang Server Operator 1.0.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:XXXX[RHBA-2023:XXXX NBDE Tang Server Operator Bug Fix Update]
+
+[id="nbde-tang-server-operator-1-0-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Lorem ipsum. (link:https://issues.redhat.com/browse/OCPBUGS-XXXXX[*OCPBUGS-XXXXX*])
```

Comment:
Greeking? Do you have finalized words for this? be careful not to let this get merged accidently until you do have the words.

Reply:
@nalhadef It should be in a block that is commented out. The intention was to prepare it for easy additions of future RNs. I can remove it if you believe that there is some danger in it.

Reply:
If it's commented out, @mjahoda, it can stay as is. Thanks!